### PR TITLE
made a change on the branch

### PR DESCRIPTION
**Pull request by aidankilleen**

### CODE REVIEW

File: src/main/webapp/aidan.jsp [hunk 1]
```diff
@@ -2,6 +2,7 @@
   <body>
     <h1> Aidans new page</h1>
     
+    <h1> Modification on aidansbranch</h1>
```
Comment (owwynne) [inline src/main/webapp/aidan.jsp]: No its not

Comment (Tuohiano) [inline src/main/webapp/aidan.jsp]: looks fishy

**Comment by leighmurph** [comment]
3th of november

File: src/main/webapp/aidan.jsp (hunk 1 above)
Comment (nettycrowley) [inline src/main/webapp/aidan.jsp]: Looks good

**Pull request merged**